app/routes.py

```diff
@@ -1 +1,237 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+import requests
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+def validate_model_by_id(cls, model_id):
```

Comment:
I like this helper function.  Could you use another to determine if the request body is correct (for post/patch requests)?